AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -74,12 +74,14 @@
     private boolean micOn;
     private boolean isVideoOnHold = false;
     private boolean isRecordingActive = false;
+    private Boolean isTranscriptionActive = false;
```

Comment:
We can just use the primitive `boolean`.

Reply:
addressed